index.src.html

```diff
@@ -293,6 +297,14 @@ <h3 id="integration-fetch">Integration with Fetch</h3>
   algorithm. That might not be a reasonable assumption, in which case we'll
   need to revisit all of this.
 
+  ISSUE: (question from palmer): What about private-to-private requests, when
+  they are mediated by the UA? For example, if the UA has loaded evil JavaScript
+  from evil-printer (192.168.0.100), and is trying to CSRF innocent-router
+  (192.168.0.1) via te browser? Although evil-printer could directly attack
+  innocent-router (e.g. port scan and then SSH password, brute-force), the
+  availability of credentials in the UA might give evil-printer more power than
+  it already has, and the UA should deny it by default. No?
```

Comment:
In the current algorithm, that would depend upon the state of the Document or Worker which initiated the request. That is, if `https://public.example.com/` loads the printer's script via `<script src="https://printer.local/script.js">` and that script initiates a request to `https://whatever.local/`, the request would be CORSified.

If, on the other hand, the user navigates to `https://printer.local/` directly (or that page allows itself to be framed or loaded as a worker), then the request to `https://whatever.local/` would work the way it does today by requiring a preflight for unsafe requests, and not requiring a preflight for "safe" requests.

It's not clear that we could change the behavior of intranets in general to require CORS when talking to other intranet servers. That seems like a bit much to ask for.